configuration/items.md

```diff
@@ -589,7 +589,14 @@ These are all in plain text, so be careful who you share these with if some data
 
 openHAB2 introduces the concept of [Things and Channels]({{base}}/concepts/things.html).
```

Comment:
Hm, I think we should stay with `{{ base }}`. Why? Because sometimes this documentation will be migrated to or used in another website on another context root path. Then it should be easy to find such magic variables and replace them with the real context root.

Reply:
You can use relative links if you prefer, `../concepts/things.html` in this case.

Reply:
Lets carve this topics out of the scope of this PR. Another PR should to this change consistently for the whole project (or at least consistently for the items.md).